meerk40t/svgelements.py

```diff
@@ -558,6 +558,7 @@ class Length(object):
     """
 
     def __init__(self, *args, **kwargs):
+        self.isValid = False
```

Comment:
isValid is camelCase and svgelements is snake_case and for various reasons it's really hard to change svgelements code since it's a stand alone project itself.